vendor/modules.txt

```diff
@@ -128,7 +128,6 @@ github.com/operator-framework/api/pkg/lib/version
 github.com/operator-framework/api/pkg/operators
 github.com/operator-framework/api/pkg/operators/v1alpha1
 # github.com/pkg/errors v0.9.1
-## explicit
 github.com/pkg/errors
```

Comment:
Shouldn't these be also removed?

Reply:
@pmacik That package is transitive dependency. For example `github.com/evanphx/json-patch` and `go.uber.org/zap` use it. Again these packages are also transitive dependencies. So, only the explicit dependency has been removed.